frame/examples/offchain-worker-ping-pong/Cargo.toml

```
@@ -0,0 +1,43 @@
+[package]
+name = "pallet-example-offchain-worker-ping-pong"
+version = "4.0.0-dev"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2021"
+license = "MIT-0"
+homepage = "https://substrate.io"
+repository = "https://github.com/paritytech/substrate/"
+description = "FRAME example pallet for offchain worker (ping-pong)"
+readme = "README.md"
```

```suggestion
readme = "README.md"
publish = false
```

Probably dont need to publish these.